Additional documentation about Open Distro for Elasticsearch Kibana reports feature

rpudlowski93 opened the pull request:
Output of the task: #1914

- Documentation added about Open Distro for Elasticsearch Kibana reports feature.

rpudlowski93 committed:
Kibana-reports-plugin-doc

atsikham commented on docs/home/howto/LOGGING.md:
```diff
@@ -175,6 +175,18 @@ PUT _template/template_01
+Since v0.10 Epiphany provide posibility to export reports from Kibana to CSV, PNG or PDF using the Open Distro for Elasticsearch Kibana reports feature.
+
+More details about the plugin and how to export reports:  
+https://opendistro.github.io/for-elasticsearch-docs/docs/kibana/reporting
```
hyperlink?

rpudlowski93 replied:
> Could you please make lines max 120 chars?

Sure, good point, I was following the standard from Elasticsearch section which is wrong so I also improved a little bit the section with elasticsearch.

atsikham commented on docs/home/howto/LOGGING.md:
```diff
@@ -175,6 +175,18 @@ PUT _template/template_01
+More details about the plugin and how to export reports:  
+https://opendistro.github.io/for-elasticsearch-docs/docs/kibana/reporting
+
+Note: Currently in Open Distro for Elasticsearch Kibana the following plugins are installed and enabled by default: security, alerting, anomaly detection, index management, query workbench, notebooks, reports, alerting, gantt chart plugins.
```
Check how other 'notes' are formatted in this document

atsikham commented on docs/home/howto/LOGGING.md:
````diff
@@ -175,6 +175,18 @@ PUT _template/template_01
 ```
 After applying this policy every new index created under this one will apply to it. There is also possibility to apply policy to already existing policies by assigning them to policy in Index Management Kibana panel.
 
+## How to export Kibana reports to csv format
````
```suggestion
## Export Kibana reports to CSV format
```

rafzei replied:
I would leave 'How to' since it's 'how to' section and we have structure like this here: https://github.com/epiphany-platform/epiphany/blob/develop/docs/home/HOWTO.md

atsikham commented on docs/home/howto/LOGGING.md:
```diff
@@ -175,6 +175,18 @@ PUT _template/template_01
 
+## How to export Kibana reports to csv format
+
+Since v0.10 Epiphany provide posibility to export reports from Kibana to CSV, PNG or PDF using the Open Distro for Elasticsearch Kibana reports feature.
```
```suggestion
Since v0.10 Epiphany provides possibility to export reports from Kibana to CSV, PNG or PDF using the Open Distro for Elasticsearch Kibana reports feature.
```

atsikham commented on docs/home/howto/LOGGING.md:
```diff
@@ -175,6 +175,18 @@ PUT _template/template_01
+
+Note: Currently in Open Distro for Elasticsearch Kibana the following plugins are installed and enabled by default: security, alerting, anomaly detection, index management, query workbench, notebooks, reports, alerting, gantt chart plugins.
+
+You can easly check enabled default plugins for Kibana using the following command on the logging machine:
```
```suggestion
You can easily check enabled default plugins for Kibana using the following command on the logging machine:
```

atsikham reviewed (changes requested):
Could you please make lines max 120 chars?

rpudlowski93 committed:
Small improvments, text formatting

rpudlowski93 commented:
> Could you please make lines max 120 chars?

Sure, good point, I was following the standard from Elasticsearch section which is wrong so I also improved a little bit the section with elasticsearch.

atsikham reviewed (changes requested):
As I remember, I removed Logstash plugin versions in ARM branch, as they are not correct after Logstash version upgrade, and there is a note with a command how to check that versions. ARM branch has been already merged.

rpudlowski93 commented:
> As I remember, I removed Logstash plugin versions in ARM branch, as they are not correct after Logstash version upgrade, and there is a note with a command how to check that versions. ARM branch has been already merged.

Sure, fixed :)

atsikham commented on docs/home/howto/LOGGING.md:
```diff
@@ -210,13 +210,25 @@ PUT _template/template_01
 policy to already existing policies by assigning them to policy in Index Management Kibana panel.
 
-## How to export Elasticsearch data to csv format
+## How to Kibana reports to CSV format
```
```suggestion
## How to export Kibana reports to CSV format
```

atsikham commented on docs/home/howto/LOGGING.md:
```diff
@@ -257,6 +269,14 @@ Elasticsearch. For non-production environment you can easily disable it by addin
 
 `/usr/share/logstash/vendor/bundle/jruby/2.5.0/gems/logstash-input-elasticsearch-*/lib/logstash/inputs/elasticsearch.rb`
 
+Check more details about configuration of input plugin in the
+[documentation](https://www.elastic.co/guide/en/logstash/current/plugins-inputs-elasticsearch.html)
+
+Check more details about configuration of output plugin in the
+[documentation](https://www.elastic.co/guide/en/logstash/current/plugins-outputs-csv.html)
+
+`Note: Currently input plugin doesn't officialy support skipping certificate validation for secure connection to Elasticsearch.`
+
```
This part is located above, lines 258-266

rpudlowski93 committed:
Typo in section name

rpudlowski93 committed:
Doc format fixed

rafzei commented on docs/home/howto/LOGGING.md:
```diff
@@ -210,13 +210,25 @@ PUT _template/template_01
-## How to export Elasticsearch data to csv format
+## How to export Kibana reports to CSV format
+
+Since v0.10 Epiphany provides possibility to export reports from Kibana to CSV, PNG or PDF using the Open Distro for
```
```suggestion
Since v1.0 Epiphany provides the possibility to export reports from Kibana to CSV, PNG or PDF using the Open Distro for
```
